package.json

```diff
@@ -48,8 +48,8 @@
   ],
   "scripts": {
     "build": "yarn build:storybook & yarn build:types & yarn build:dist",
-    "build:storybook": "cross-env NODE_ENV=production && rimraf public/dist && storybook build -o public/dist",
-    "build:dist": "cross-env NODE_ENV=production && vite build",
+    "build:storybook": "rimraf public/dist && NODE_ENV=production storybook build -o public/dist",
```

Comment:
**note:** I doubt we need to set `NODE_ENV` to production when running `storybook build`, but I've left it here just in case. I know it's not necessary for `vite build` below, because Vite's build command implicitly builds for production unless explicitly told otherwise.